src/features/game/components/bank/components/Withdraw.tsx

```
@@ -80,53 +84,90 @@ type Page =
+    <div className="flex flex-col gap-2 p-1">
+      <span className="text-xs">{t("withdraw.menu.intro")}</span>
+
+      {/* FLOWER token */}
+      <ButtonPanel
+        onClick={() => setPage("tokens")}
+        className="flex items-center justify-between"
+      >
+        <div className="flex items-center gap-2">
+          <img src={flowerIcon} className="w-8 h-8" />
+          <div className="flex flex-col">
+            <span className="text-sm">FLOWER</span>
```

ESLint

[error] Missing JSX expression container around literal string: "FLOWER"

(react/jsx-no-literals)